README.md

```diff
@@ -16,418 +15,66 @@ Sorting Hat is a part of the [GrimoireLab toolset](https://grimoirelab.github.io
-```
-
-You are ready to use sortinghat!
-
-### Docker
-
-You can use our image from [DockerHub](https://hub.docker.com/r/grimoirelab/sortinghat/) (`grimoirelab/sortinghat`) and skip the `docker build` step.
-If you prefer to build the image yourself execute:
-
-```sh
-$ docker build -t grimoirelab/sortinghat .
-```
-
-Next step would be to start a MySQL docker container for data storage:
-
-```sh
-$ docker run --name mysql \
-             -e MYSQL_ROOT_PASSWORD=sortinghat \
-             -d mysql
-```
-
-Run the sortinghat docker container in interactive mode:
-
-```sh
-$ docker run -i -t --rm \
-             --link mysql:mysql \
-             -e SORTINGHAT_DB_HOST=mysql \
-             -e SORTINGHAT_DB_PASSWORD=sortinghat \
-             -e SORTINGHAT_DB_DATABASE=sortinghat \
-             grimoirelab/sortinghat \
-             /bin/bash
-```
-
-Now you can initialize sortinghat with the database name `sortinghat`:
-
-```
-$ sortinghat init sortinghat
-```
-
-You are ready to use sortinghat and explore the commands documented below.
-Have fun!
-
-## Configuration
-
-Set the database parameters via the `config` command:
-
-```
-  $ sortinghat config set db.host <mysql-host>
-  $ sortinghat config set db.user <user>
-  $ sortinghat config set db.password <password>
-  $ sortinghat config set db.database <name>
-  $ sortinghat config set db.port <port>
-```
-
-Alternatively you can set environment variables:
-
-```
-  $ export SORTINGHAT_DB_HOST=<mysql-host>
-  $ export SORTINGHAT_DB_USER=<user>
-  $ export SORTINGHAT_DB_PASSWORD=<password>
-  $ export SORTINGHAT_DB_DATABASE=<name>
-  $ export SORTINGHAT_DB_PORT=<port>
-```
-
-After this initialize a new database:
+### From source code
 
+Clone the repository, and change to the `muggle` branch
 ```
-  $ sortinghat init <name>
+$ git clone https://github.com/chaoss/grimoirelab-sortinghat
+$ git checkout muggle
 ```
 
-## Compatibility between versions
+#### backend
 
-Python 2.7 is no longer supported. Any code using this version will
-not work. Please update your code to 3.4 or newer versions.
-
-SortingHat databases previous to 0.7.0 are compatible but UTF-8 4-bytes
-characters will not be inserted in the database and causing errors. For this
-reason, it is recommendable to update its schema. The fastest way is to
-dump the data into a file, regenerate the database with `init` command
-and restore the data from the dump.
-
-SortingHat databases previous to 0.6.0 are no longer compatible.
-The database schema changed in `profiles` table to add the fields `gender`
-and `gender_acc`.
-
-The next MySQL statements should be run to update the schema
+We use [Poetry](https://python-poetry.org/docs/) for managing the project.
 
+Install the required dependencies, we recommend to set up a Python virtual environment to run it
 ```
-mysql> ALTER TABLE profiles ADD COLUMN gender VARCHAR(32) DEFAULT NULL
-mysql> ALTER TABLE profiles ADD COLUMN gender_acc INT(11) DEFAULT NULL
+$ pyenv virtualenv 3.6.13 sh_venv
```

Comment:
I think that instead of running `pyenv`, users can run `poetry install` directly and then run `poetry shell`. If I'm right, `poetry install` creates a python environment for the project that will be activated running `poetry shell`.

Reply:
Yeah, what you have said is correct. But, I was considering using only python 3.6 for now since we have only this in the CI.

But, I think you are right. I will update the PR.